### ISSUE TITLE
JWST asset needs to be separated into multiple smaller assets

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3425]
The [James webb asset](https://github.com/OpenSpace/OpenSpace/blob/master/data/assets/scene/solarsystem/telescopes/jwst/jwst.asset) includes a lot more than just the model of the James Webb telescope (which is what I would have expected).

It seemingly loads everything we have related to the James Webb mission, including the Band, the FoV of the telescope, multiple scene graph nodes showing L2, and the actions and mission file. These should all be split into individual assets, so a user can add only the part they want. 

All these individual parts are also enabled per default, which means that if I want to show only the model, I have to first disable all the stuff that I don't want/need in my scene, which is very annoying. 

Short version: 
* There is no way to add just the JWST model. There should be.

**Comment by WeirdRubberDuck** [comment]
The L2 parts are extra confusing because they are being added as part of the transforms asset for JWST. This to me should mean that only the transforms for L2 should be included, but it actually also adds the visual elements. 

Figuring out why those visuals were added, just form adding the jwst.asset file, was a bit cumbersome

**Comment by alexanderbock** [comment]
The `jwst.asset` should load everything, but I definitely agree that there shoudl be a `model.asset` that only does the JWST model itself.   We basically ended up with a naming scheme of `folder-name/folder-name.asset` as the "main" asset for a folder
